src/Swarm/TUI/Controller.hs

```diff
@@ -79,7 +79,9 @@ import Swarm.Game.Robot
 import Swarm.Game.ScenarioInfo
 import Swarm.Game.State
 import Swarm.Game.Step (finishGameTick, gameTick)
-import Swarm.Language.Capability (Capability (CDebug, CMake))
+
+-- See Note [liftA2 re-export from Prelude]
```

Comment:
I think this comment got moved inadvertently by `fourmolu`, perhaps?

Reply:
Yeah it was some fail, I got two restyled PRs in succession. Will take a look, maybe it’s something we could report to fourmolu.